corehq/apps/userreports/app_manager/helpers.py

```diff
@@ -37,7 +37,7 @@ def get_case_data_source(app, case_type):
         table_id=clean_table_name(app.domain, case_type),
         display_name=case_type,
         configured_filter=make_case_data_source_filter(case_type),
-        configured_indicators=meta_indicators + dynamic_indicators,
+        configured_indicators=meta_indicators + dynamic_indicators + _get_shared_indicators(),
```

Comment:
this is a small change to current behavior, adding a `count` column to all autogenerated (from apps) form and case tables.

Reply:
Good question. I added it because I wanted something to `sum` and because I'd seen it used widely in other UCRs. But now that you ask I think it may not be necessary if we have a `count` operator - which I'm definitely planning to add to this framework.

Reply:
Yeah. I think that this is one of those common UCR workflows that only exists because we don't have a reasonable way to do this. https://github.com/dimagi/commcare-hq/pull/20452 is somewhat related (what you currently need to do to count on different sets of filters)